producers/smd_producer.py

```diff
@@ -286,24 +343,34 @@ def global_except_hook(exctype, value, exc_traceback):
 def get_xtc_files(base, hutch, run):
     """File all xtc files for given experiment and run"""
     run_format = ''.join(['r', run.zfill(4)])
-    data_dir = ''.join([base, '/', hutch.lower(), '/', exp, '/xtc'])
-    xtc_files = glob(''.join([data_dir, '/', '*', '-', run_format, '*']))
+    data_dir = Path(base) / hutch.lower() / exp / 'xtc'
+    xtc_files = list(data_dir.glob(f'*{run_format}*'))
+    logger.debug(f'xtc file list: {xtc_files}')
     return xtc_files
 
 def get_sd_file(write_dir, exp, hutch):
     """Generate directory to write to, create file name"""
+    print(exp)
+    print(hutch)
```

Comment:
Are these meant for production or overlooked debug lines?

Reply:
Debug lines!